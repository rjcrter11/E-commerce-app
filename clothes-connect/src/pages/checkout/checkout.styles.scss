.checkout-greeting {
    width: 100%; 
    text-align: center;
    font-weight: bold;
    font-size: 24px;
    font-family: 'Open Sans Condensed';
    color: rgb(85, 84, 84);  
}

.checkout-page {
    width: 55%; 
    min-height: 90vh;
    display: flex;
    flex-direction: column;
    align-items: center;
    margin: 50px auto 0; 
    font-family: 'Open Sans Condensed';
    font-size: 20px; 

    .checkout-header {
        width: 100%; 
        padding: 10px 0; 
        display: flex;
        justify-content: space-between;
        border-bottom: 1px solid darkgray;

        .header-block {
            text-transform: capitalize;
            width: 23.5%; 

            &:last-child {
                width: 8%; 
            }
        }
    }

    .total {
        margin-top: 30px; 
        margin-left: auto; 
        font-size: 36px;
    }

    button {
        margin-left: auto;
        margin-top: 50px; 
    }

    .test-warning {
        text-align: center;
        margin-top: 40px; 
        color: red; 
        font-size: 20px; 
    }
}