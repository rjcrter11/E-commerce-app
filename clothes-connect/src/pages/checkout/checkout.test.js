import React from 'react';
import { create } from 'react-test-renderer';
import { CheckoutPage } from './checkout.component';


let wrapper;

beforeEach(() => {
    const mockProps = {
        cartItems: [],
        total: 100
    };
    wrapper = create(<CheckoutPage {...mockProps} />)
});

it('should render CheckoutPage component', () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
})

