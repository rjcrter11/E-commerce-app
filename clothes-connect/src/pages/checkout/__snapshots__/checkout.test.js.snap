// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CheckoutPage component 1`] = `
<div
  className="checkout-page"
>
  <div
    className="checkout-header"
  >
    <div
      className="header-block"
    >
      <span>
        Product
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Description
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Quantity
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Price
      </span>
    </div>
    <div
      className="header-block"
    >
      <span>
        Remove
      </span>
    </div>
  </div>
  <div
    className="total"
  >
    <span>
       TOTAL: $
      100
       
    </span>
  </div>
  <div
    className="test-warning"
  >
    *Please use the following test credit cart for payments*
    <br />
    4242 4242 4242 4242 - Exp: 12/20 - CVC: 123
  </div>
  <button
    className="StripeCheckout"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    onMouseOut={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "background": "linear-gradient(#28a0e5,#015e94)",
        "border": 0,
        "borderRadius": 5,
        "boxShadow": "0 1px 0 rgba(0,0,0,0.2)",
        "cursor": "pointer",
        "display": "inline-block",
        "overflow": "hidden",
        "padding": 1,
        "textDecoration": "none",
        "userSelect": "none",
        "visibility": "visible",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundImage": "linear-gradient(#7dc5ee,#008cdd 85%,#30a2e4)",
          "borderRadius": 4,
          "boxShadow": "inset 0 1px 0 rgba(255,255,255,0.25)",
          "color": "#fff",
          "display": "block",
          "fontFamily": "\\"Helvetica Neue\\",Helvetica,Arial,sans-serif",
          "fontSize": 14,
          "fontWeight": "bold",
          "height": 30,
          "lineHeight": "30px",
          "padding": "0 12px",
          "position": "relative",
          "textShadow": "0 -1px 0 rgba(0,0,0,0.25)",
        }
      }
    >
      Pay Now
    </span>
  </button>
</div>
`;
